Grades.java

```diff
@@ -1,18 +0,0 @@
-/**
- * Write a program that will compute some statistics about students grades.
- * It will ask the user to enter the number of students.
- * Then, the program will ask the user to enter the grades one by one.
- * The range of each grade is an integer between 0 and 100.
- * Make sure to use an array where the type is the closest to this range (hint: don't use an array of "int"'s).
- * If the user enters a number which is not in range, the program will ask him to enter a new grade.
- * Once the user entered all the grades, the program will print:
- * 1. The highest grade and its index (something like: the student #10 got the highest grade and it's 98).
- * 2. The lowest grade and its index (something like: the student #3 got the lowest grade and it's 13).
- * 3. The average of the grades
- */
-public class Grades {
-
-    public static void main(String[] args) {
-        // enter your code here :)
-    }
-}
```

Comment:
@ameedf Please ignore the clutter... had to send this pull request from my work laptop, without my git repo.
Thanks!